<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>112</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Height(cm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>153</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight(Kg)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btmCalculate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>89</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BMI calculator</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reslut</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>71</y>
      <width>240</width>
      <height>147</height>
     </rect>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Underweight</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Normal</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Overweight</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>BMI</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>&lt;18.5</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>18.5 - 22.9</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>&gt; 22.9</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_height">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Weight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Weight</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHello_World">
    <property name="title">
     <string>BMI Calculator</string>
    </property>
   </widget>
   <addaction name="menuHello_World"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
